papis/commands/update.py

```diff
@@ -464,7 +464,7 @@ def run(
           papis.cli.FormattedStringParamType(),
           papis.cli.FormattedStringParamType()),
 )
-@papis.cli.bool_flag("-b", "--batch", help="Batch mode, do not prompt or otherwise")
+@papis.cli.bool_flag("-b", "--batch", help="Batch mode, do not prompt and do not abort on error.")
```

Comment:
Is this accurate?

Reply:
I don't think so. `--batch` is mostly meant to be used in scripts so that the command doesn't start blocking and asking for user input, but errors should still be reported in the same way.

Maybe `Batch mode (do not block on user input)` or something like that?

Reply:
I took this from [here](https://papis.readthedocs.io/en/latest/commands/update.html#module-papis.commands.update):

> Normally, `papis update` will abort on encountering an error. If you want to skip errors and apply as many changes as possible, use the `--batch` flag.

Meaning I think it does still *report*, just doesn't stop execution on the first error?

Given the above, should we still change this?

Reply:
Hm, good point, forgot about that! I was thinking about `papis add`.. 

In that case, maybe `Batch mode: do not prompt and skip documents with errors`?

Reply:
Yeah, sounds good